packages/core-contracts/contracts/mocks/utils/MathUtilMock.sol

```diff
@@ -7,4 +7,44 @@ contract MathUtilMock {
     function sqrt(uint x) public pure returns (uint z) {
         return MathUtil.sqrt(x);
     }
+
```

Comment:
This is introducing code duplication. If its a mock that simply tests the MathUtil library, it should simply provide wrapper functions that expose the library's internal functions.

Reply:
this code was from @leomassazza because this merge from `feature-v3-mvp` to main includes all the code that leo worked on previously. @leomassazza want to comment on this?